src/plugins/intel_gpu/src/graph/deconvolution.cpp

```diff
@@ -82,16 +84,28 @@ layout deconvolution_inst::calc_output_layout(deconvolution_node const& node, ke
                              "As for now, deconvolutions with more than 3 dimensions are not supported");
 
     int32_t x = static_cast<int32_t>(
-        off_factor * pad[pad.size() - 1] + (input_layout.spatial(0) - 1) * strd[strd.size() - 1] + weights_layout.spatial(0));
+        off_factor * pads_begin[pads_begin.size() - 1] +
+        off_factor * pads_end[pads_end.size() - 1] +
+        output_padding[output_padding.size() - 1] +
```

Comment:
Why do we need take output_padding into consideration here? I think `x` is supposed to be the size of actual data without any paddings